detailed_docs/Wiki/0249_capabilities.md

```diff
@@ -0,0 +1,135 @@
+## Feature 1: As a mobile application, I want to get HMI capabilities from SDL after Ignition OFF/ON cycle so that I can interact with the HMI efficiently
```

Comment:
It's better to use an "as a Client..." instead of "mobile application". In case if we have any specific case, we need to describe it as a separate feature.

Reply:
The proposal didn't say this way of retrieving of HMI capabilities is applicable for all kind of applications. That's why I assumed it was related to mobile apps.
Please, confirm that the flow for obtaining capabilities for web_engine and cloud applications is identical and all of the 'features' listed here are applicable for the mentioned apps.